README.md

```diff
@@ -2,7 +2,7 @@
 
 <p align="center">
  <h2 align="center"><strong>@lukso/lsp-factory.js</strong></h2>
- <p align="center">Helper Library to allow simple deployments of <a href="https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-3-UniversalProfile.md">LSP3UniversalProfiles</a> and <a href="https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-4-DigitalCertificate.md">LSP4DigitalCertificates</a>.</p>
+ <p align="center">Helper Library to allow simple deployments of <a href="https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-0-ERC725Account.md">UniversalProfiles</a> and <a href="https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-4-DigitalCertificate.md">LSP7</a> and <a href="https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-8-IdentifiableDigitalAsset.md">LSP8 </a>Digital Assets.</p>
```

Comment:
Maybe you can keep `LSP3` in next to Universal Profiles? 

Package name is `lsp-` and you wrote LSP7, LSP8, so let's add LSP3 too?